docs/guides/serverless-run-first-workload.ipynb

```diff
@@ -0,0 +1,384 @@
7XYfNCO2fm0w/PYl3+zwY+dJAM+/8EsenN2Xrv54ZIc/91OGfentKpuW+oYAI1WqveTUvP6DQCw8Wnp56bOQQFYk4Z0KO/FFWTQjGa+MC6PvTCuZMcf+etbS3ZsAAAAAADYWLkHGQAAAAAAAGVFQQYAAAAAAEBZUZABAAAAAABQVhRkAAAAAAAAlJWqUgdg/bStTB4+utQpmk7byubZb1W7Njll3J+aZ+ess6p2bUodAQAAAAAAFGSbukIhaee7uFaFQiGtqtuWOgYAAAAAALARMMUiAAAAAAAAZUVBBgAAAAAAQFlRkAEAAAAAAFBWFGQAAAAAAACUFQUZAAAAAAAAZUVBBgAAAAAAQFlRkAEAAAAAAFBWFGQAAAAAAACUFQUZAAAAAAAAZUVBBgAAAAAAQFlRkAEAAAAAAFBWFGQAAAAAAACUFQUZAAAAAAAAZUVBBgAAAAAAQFlRkAEAAAAAAFBWFGQAAAAAAACUFQUZAAAAAAAAZaWq1AEAAAAAAJpCsZgsrit1ioZpW5kUCqVO0XJsaj8Dvv9QOgoyAAAAAKBFWFyXvO/OUqdomIePTtr5lLbJbGo/A77/UDqmWAQAAAAAAKCsKMgAAAAAAAAoKwoyAAAAAAAAyoqCDAAAAAAAgLKiIAMAAAAAAKCsKMgAAAAAAAAoKwoyAAAAAAAAyoqCDAAAAAAAgLKiIAMAAAAAytqM+6/JMx8qZMb916x2/ZLp4/PMhwoZ//MzNmguAJqPggwAAAAAAICyoiADAAAAAACgrCjIAAAAAACgEWbcf01e/dohpY4BNIKCDAAAAAAAgLJSVeoAAAAAAACwKRn/i7My+5E/p1hXm2Ldsjz30Q5JksHXz02hsrLE6YB1UZZXkM2YMSPDhg3LTjvtlLZt22bbbbfN+eefn5qampx55pkpFAq54oorSh0TANjUFArpf/ax+fDDP89pr1+fk57+Tfa6+JOpatem1MkAACg3zk2hQZZMey1v/PLsvPSFXfLsSdV5/hNdMvKcfnn9Z6dn/r8fWGV8ry/+LrsPX5DtPv+bdOj/vuw+fEF2H75AOQabkLK7guz555/PUUcdlWnTpqV9+/bp379/pkyZkssvvzzjxo3LrFmzkiSDBw8ubVAAYJOzz3fOSP+zjskbdz6Rl35zWzr32Sb9zzw6m+/aO3ef/J2kWCx1RAAAyoRz0+ZSWPGvWY/cmDdvvzyLXn8+VZ26Zbffji9dLNZLzZinM/pr70+hqlW6HvLJtNtuQOqXLsqSKWMy7/l7UtmuYzoOHFLqmEATK6uCbMaMGTnuuOMybdq0XHDBBbn44ovTsWPHJMmPfvSjXHTRRamqqkqhUMjAgQNLnBYA2JR07tsz/T59VMbf8a88eNZlK5bPn/Bm9vv+mel9/IF5/ZZHSpgQAIBy4dy04Spat0uS1C9ZuNr19Utqlo9r027FsqoOXbLl0edm2dzpefPWnzZ/SJrN1OHfTv2Shel36fOp7j1olfXLZk8rQSqguZXVFIvnnXdeJk2alHPPPTeXXXbZinIsSYYNG5ZBgwaltrY2vXr1SqdOnUqYFADY1PT+8EEpVFTk5d/esdLyMdfdl2ULF2fHEw4uUTIAAMqNc9OGa7NV7yTJ4kmjVrt+8cTly1tv2XvFsk6DD0/Xgz+WNlts3/wBaVaLp4xJZcfNV1uOJUmrLt03cCJgQyibgmzUqFEZPnx4unXrlksuuWS1Y/bcc88kyaBB/3khfLtQ22effdKmTZsUCoXVbpsk9913X/bbb7+0bds2W265ZT73uc9l7ty5TftEAICNUrfBO6W+ri4znhuz0vK6Jcsy66Xx6TZ4xxIlAwCg3Dg3bbjqHfdIq27bZvbDf87SmVNWWle/bGnevPOKpFBI530+WKKENKc23XdM3fyZmf34X0sdBdiAyqYgu+GGG1JfX59TTjklHTp0WO2Ydu2WXyL9zoJs7Nixufnmm9O9e/fsvffea9z/Qw89lCOPPDLbbLNNbrnllnz/+9/PTTfdlOOPPz5FczoDQItXvVWXLJk1P/VLa1dZt3DarLTdfLNUtCqr2a0BACgR56YNV6isyvaf/3XqFs7Ny+cPzKT/vShv3X1Vpg7/bkb99x5Z8NJD6X7CV9K2586ljkoz6HHy11OoapXXfnhCXvp834y//NN56x+/zqKJq7+iEGgZyuadcMSIEUmSIUPWfDPFSZMmJVm5IDv44IMzderUJMm3vvWtPProo6vd9jvf+U769OmTv/zlL6moWN47br755jnhhBNyxx135Nhjj21U7j59+qzYHwBQWq2KFbk4+6x2XWW7Nqlbumy16+qWLF9e1a51li5b9UOKjV3fPn2zrFBf6hgAALxDSz83bew5aKF1u2z1szFrH7gam+11THb+4aOZ9tcfZeaI/03t/JmpbNs+7Xrvnt4XDk/Xg05u1H7Xpm/fPikuXdQs+y5HjfkZ6LDL/un3k2cy/e8/ydxn/pGZ9/8hM+//w/J1/d+XXudfkzbdd1j98SpbrbiHXWP4/kPD1df/5/3hoIMOynPPPdeo/ZRNQfbGG28kSbbffvVzAtfW1q4ov95ZkK1rOfXEE0/kU5/61ErjjzjiiCTJ3/72t0YXZG+XcwBA6bUuVCZbrX5d3aIladV+s9Wuq2zTKklSu2hpc0VrVlOmTsnSYl2pYwAA8A4t/dy0seegFW2q1/RlWSft++ydHS/6y3rsoeGmTJmS+iULN+gxW7LG/gy067Vbep1/TZJkyZtvZMFLD2XGvb/LgpcfztgffCj9fvJMKlq1XmW7zQ85JZsfckqj8/r+w/qZPn16o7ctm4KspqYmSbJo0erb+OHDh2fGjBnp2LFjevfuvdox76WysjKtW6/8AtmqVasUCoWMHDmy4YH/T48ePVxBBgAbiVbFimQNf8S6cPrsbNa3ZypaV60ylU11965ZPHNu6jfiv9B9L1v32NoVZAAAG5mWfm7a2HPQwnpcydNQxbq6FOuWpVi3LCkWU
```

Comment:
@pandasa123 please verify I've preserved the meaning in this line - is it true that the queue priority is based on usage?